configuration/migration/index.md

```diff
@@ -46,6 +46,8 @@ You should probably do your usual OS backup too to make sure you can get back to
 
 openHAB (since version 4) requires Java 17. Most modern Linux OS have an `openjdk-17-jdk` package available for you to install but alternative solutions such as [Zulu's OpenJDK](https://www.azul.com/downloads/zulu-community/?version=java-17-lts&package=jdk) (which openHABian can install for you) are available and these may run faster on 32-bit armhf platforms such as the Raspberry Pi.
 
+Upcoming openHAB 5 development requires Java 21.
```

Comment:
Maybe it would be better to not use the word "upcoming" so we don't have to come back to this later when it's no longer upcoming? And I guess also "development" can be omitted, since this page is about migration? Looking at the sentence above, maybe:

```suggestion
openHAB 5 requires Java 21.
```

?

(the sentence above regarding Java 17 for openHAB 4 might also need an update, but this is unrelated to your changes)